itg.f90

```diff
@@ -52,6 +53,12 @@ PROGRAM itg
 
   eta = etag * ( 1.0_dp - etac * (xx/nqp)**2 )
 
+!--------------------------!
+! Create runpath directory !
+!--------------------------!
+
+  CALL SYSTEM('mkdir ' // runpath)
```

Comment:
I didn't include a check for the directory already existing, because we should be changing the `runpath` between runs, and I'd prefer to see an error if we forgot to change it (as opposed to overriting each run accidentally).

Reply:
@EdwardSimmons  I think it would not stop the run though and continue, i.e. if it finds the directory already, it'll simply go ahead anyway, writing to it?